### PR TITLE
Default fee reduction in Qt

### PR DESCRIPTION
**Pull request by riordant**
Change default fee per byte from 100 sat/byte to 1 sat/byte in Qt.

Fixes https://github.com/zcoinofficial/zcoin/issues/233

### CODE REVIEW

**Comment by ultimaweapon** [comment]
We should edit the PR description to automatically close the issue when it merge. This will also make it immediately see the source issue too.

**Comment by riordant** [comment]
I think it already does this @ultimaweapon, After I merged your PR related to TOR build yesterday it closed related issue automatically.

**Comment by ultimaweapon** [comment]
Nope. You need to put a special keyword in the PR description. See https://help.github.com/en/articles/closing-issues-using-keywords for how to do it.

**Comment by riordant** [comment]
Ok, well it definitely did it in that case at least.

**Comment by reubenyap** [comment]
Any other outstanding issues on this?

**Comment by ultimaweapon** [comment]
@reubenyap right now Sigma team is quite busy for beta release so we need somebody to review this.

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -103,7 +103,7 @@ SendCoinsDialog::SendCoinsDialog(const PlatformStyle *platformStyle, QWidget *pa
         settings.setValue("nTransactionFee", (qint64)DEFAULT_TRANSACTION_FEE);
     if (!settings.contains("fPayOnlyMinFee"))
         settings.setValue("fPayOnlyMinFee", false);
-    ui->customFee->setValue(CENT/10);
+    ui->customFee->setValue(CWallet::GetRequiredFee(1000));
```
[the remark this answers is not in the document]
Comment (riordant) [reply src/qt/sendcoinsdialog.cpp]: Yes I agree. I will set it to a hard value

**Comment by riordant** [comment]
Updated and squashed as one commit

**Pull request merged**